app/screens/recent_mentions/recent_mentions.js

```diff
@@ -224,8 +224,8 @@ export default class RecentMentions extends PureComponent {
 
         return (
             <SafeAreaView
-                style={style.container}
                 testID='recent_mentions.screen'
+                style={style.container}
```

Comment:
why this change? if anything it should be alphabetical order.. `s, t` no need to change on this PR but take into account for future changes.